src/collection-facets.ts

```diff
@@ -321,12 +335,58 @@ export class CollectionFacets extends LitElement {
           ${this.collapsableFacets ? collapser : nothing} ${facetGroup.title}
         </h1>
         <div class="facet-group-content ${isOpen ? 'open' : ''}">
-          ${this.getFacetTemplate(facetGroup)}
+          ${this.getFacetTemplate(facetGroup)} ${this.getMoreLink(facetGroup)}
         </div>
       </div>
     `;
   }
 
+  /**
+   * Generate the More... link button just below the facets group
+   */
+  private getMoreLink(facetGroup: FacetGroup): TemplateResult | typeof nothing {
+    // don't render More... link if you facets is < 5
+    if (Object.keys(facetGroup.buckets).length < 5) return html``;
+
+    return html`<button
+      href="javascript:void(0)"
+      class="more-link"
+      @click=${() => {
+        this.emitMoreLinkClickedEvent(facetGroup);
+      }}
+    >
+      More...
+    </button>`;
+  }
+
+  async emitMoreLinkClickedEvent(facetGroup: FacetGroup) {
+    const config = new ModalConfig();
```

Comment:
suggestion add properties into constructor as args

```suggestion
    const config = new ModalConfig({
      headline,
      config.closeOnBackdropClick: true
    });
```